### ISSUE TITLE
Bug report: Shadows in RTL view

### ISSUE THREAD

**Issue by ShlomoCode** [localsend/localsend#795]
### Describe the bug

When the display is in an RTL language, such as Hebrew or Arabic, the silhouette displayed when hovering over a tab in the application appears in the wrong place

### To reproduce

Change the app language to Hebrew or Arabic

### Expected behavior

The shadows will be displayed properly, under the tab label

### Screenshots

https://github.com/localsend/localsend/assets/78599753/e78b4428-c98a-4407-b92d-8c6b847d230d

### Desktop (please complete the following information)

- OS: MacOS
- Version: 1.11.1 (35)

**Comment by Tienisto** [comment]
Thanks. I think this has been noticed in #731 already. Can you open an issue in https://github.com/flutter/flutter?

There is nothing we can do on the LocalSend side because those are just plain Flutter widgets

**Comment by ShlomoCode** [comment]
> Can you open an issue in https://github.com/flutter/flutter?

Absolutely, could you just give me more technical details about the problematic widget so I can write there? I don't know Flutter

**Comment by Tienisto** [comment]
It's a `NavigationRail`

**Comment by ShlomoCode** [comment]
@Tienisto They demand "Please create a minimal reproducible sample that shows the problem" :|

**Comment by Tienisto** [comment]
No worries, I will open the issue for you!

**Comment by ShlomoCode** [comment]
@Tienisto just a reminder

**Comment by Tienisto** [comment]
I just check out the latest Flutter version (3.16.8) and they seem to have fixed it.
